### ISSUE TITLE
Archetype has been sunset

### ISSUE THREAD

**Issue by kgiszewski** [kgiszewski/Archetype#439]
It's with mixed feelings that I announce that Archetype is now sunset. The final version released is v1.18.0 which went out today.

## Thank YOU ALL

Thank you ALL for the hard work and patience with this project. A super-amazing-mega-ultra #h5yr to all the contributors. And of course thanks to all who discovered, used and enjoyed Archetype.

A personal thank you to:

@kjac 
@tomfulton 
@leekelleher 
@Nicholas-Westby 
@mattbrailsford 

You guys really did amazing work 👍 

## So why sunset?

- We've had a good run, but the ROI of maintaining this project has waned
- Nested Content is core as-of v7.7.x, people can\should use that for most of what Archetype did

## So now what?

I have removed everyone from having commit rights. If anyone wants to carry on this project, I won't stop you so long as you do the following:

1. Fork the project
2. Rename it from Archetype to a name of your choosing
3. Do not use the logo

You can release it under any license you want. The name 'Archetype' is special and therefore should not be used in any forks.

Thanks again everyone.

**Comment by jonrandahl** [comment]
Kevin, and everyone, 

Thank you for all the work on Archetype as it's helped me build many a wonderful site for clients and I truly appreciate everything you all have done. 

Most sincerely,
Jon

**Comment by leekelleher** [comment]
...and **thank you** @kgiszewski! #h5yr

**Comment by rbaty-barr** [comment]
yes, a hearty thanks to all. It is projects like this that help a community learn and also inspire others to create great things. Archetype is definitely one of the projects that have moved our community and the umbraco product into it's next evolution. #H5YR

**Comment by jesperordrup** [comment]
Thank you to all on the Archetype team. This package solved many complicated implementations for us.  

#H5yr

**Comment by DanDiplo** [comment]
A big thank you to Kevin and everyone who has contributed to this project. I can honestly say it's been one of the most useful packages I've used.

I also know how difficult it can be supporting packages. It's a lot of work, work that you don't get paid for and time you don't get back.

But I am a bit puzzled as to why you aren't letting other people commit to this repo? It's fair enough saying, "fork it" under another name but that will lead to fragmentation and probably break any upgrade paths. So if a tiny bug is found in future then it's making it unfixable. And I can't really understand why?

**Comment by kgiszewski** [comment]
@DanDiplo Thx for the love and questions.

I understand your curiousness about the source.

The amount of grief\work\effort that comes with running an OSS package takes a toll. Especially when it is unfunded or more accurately, funded by the time of the contributors and their families. I know I'm preaching to the choir with respect to you specifically.

Fragmentation may happen.

There are other examples out in the wild where a similar approach has been taken. For instance NPoco is the next generation PetaPoco which I believe began as a continuation fork.

Committing new code sounds easy enough, but is just one aspect. Who will create the build? Who will manage the releases on Nuget\Our?

I want to avoid a cycle of "just one more fix". If there is a minor bug, simply fork it and fix it locally. If the core introduces a breaking change, perhaps someone will continue this project as a renamed fork. 

Quite simply, this project is out of business. Trust me, I've tried my ass off to try to make this project work financially. I've ran this project almost 4 yrs and it's time to move onto something else. I do not wish to compete with the core version of Nested Content which I feel people should start moving towards if they have not already.

Again thanks for your support.

**Comment by sniffdk** [comment]
Thank you @kgiszewski, it was a good run for sure, I'm sending much love and respect and many many #h5yr's your way. So Long, and Thanks for All the Fish 😄

**Comment by DanDiplo** [comment]
Hey, @kgiszewski  I understand now. I was naively thinking you could just somehow hand the project over to someone and they'd take over, but I forgot about all the other stuff (not sure how I did that, as I'm well aware of _that_ pain, lol). I guess you can't just transfer ownership of a repo and package that easily, eh? 

I was more concerned with maintaining backward compatibility, so an old version could be upgraded to a potential newer version. But I guess if the data format (JSON) stays the same then a new package could be created and the datatype swapped without loss. So it should be possible.

Anyway, thanks for all the hard work - really appreciated!

**Comment by kgiszewski** [comment]
@DanDiplo Sorry for the wall of text in my last response :)

**Comment by Eudaemonic** [comment]
@kgiszewski a deeply intelligent and inspiring piece of work. #h5yr

**Comment by alanmac** [comment]
Congrats on creating and maintinaing such a well thought out and loved product for the community Kevin. It's clear to see the impact that it's had on Umbraco's evolution and no doubt brought tons of end users lots of joy.  :clap::clap::clap:

**Comment by JimBobSquarePants** [comment]
You pushed the boundaries with this plugin and showed the community a new level to aim for. 

You should be immensely, and rightfully proud. 🥇

**Comment by johnseto** [comment]
Sad to see it go, I used it in the early days before Nested Content came along. Thanks very much for producing this package.

**Comment by gordonsaxby** [comment]
Have used ArcheType many many times and really liked it, so sad to see it go. A massive thanks for all the work you put into it.

**Comment by JFSeibert** [comment]
Just read about Archetypes sunset and it hurts a bit. This piece of software quite often saved my a**.
Big hugs and thank you for the great work you've done.

**Comment by nathanwoulfe** [comment]
Realllllllly late to the party, but want to echo all the above sentiments. Awesome package, shame to see it go.
